demos/interactive_face_detection_demo/face.cpp

```diff
@@ -17,13 +11,15 @@ Face::Face(size_t id, cv::Rect& location):
 }
 
 void Face::updateAge(float value) {
+    _isAgeGenderEnabled = true;
```

Comment:
So, again, why do we need it here? Please come up with a detailed explanation on how it relates to the rest of the app.

Reply:
The `visualizer` needs to know what to render for correct rendering
Now fields like `_isAgeGenderEnabled` are visualizer fields.
Is this a valid implementation?